packages/react-router-config/README.md

```diff
@@ -134,7 +139,7 @@ class PendingNavDataLoader extends Component {
     previousLocation: null
   }
 
-  componentWillReceiveProps(nextProps) {
+  componentDidUpdate(nextProps) {
```

Comment:
The signature is prevProps for cDU, so some of the code below needs to be updated to match. Basically s/this.props/prevProps/ and s/nextProps/this.props/.

Reply:
Did you mean to swap it like in here: [react docs](https://reactjs.org/docs/react-component.html#componentdidupdate)?

As in, put `this.props` first: `if (this.props.location !== nextProps.location)`

rather then: `if (nextProps.location !== this.props.location)`

or keeping it in the current order is fine?

Reply:
Ok, I made sure to do the swap anytime a value is set based upon `prevProps` or `this.props`. Hope we're good now! And sorry about not doing the most thoughtful PR! Especially at the beginning, I was a bit confused seeing the result of prettier and reverted my commits a few times resulting in a bit of frustration each time I rewrote the code.